docs/Noise.ipynb

```diff
@@ -0,0 +1,467 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Using Gaussian"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To use the Gaussian mechanism for adding noise, we specify a different parameter, `rho`, instead of `epsilon`. Similar to how `epsilon` is usually set to one, `rho` is usually set to 0.5. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<div><style>\n",
+       ".dataframe > thead > tr,\n",
+       ".dataframe > tbody > tr {\n",
+       "  text-align: right;\n",
+       "  white-space: pre-wrap;\n",
+       "}\n",
+       "</style>\n",
+       "<small>shape: (4, 2)</small><table border=\"1\" class=\"dataframe\"><thead><tr><th>ILOSTAT</th><th>Gaussian Noise</th></tr><tr><td>i64</td><td>u32</td></tr></thead><tbody><tr><td>3</td><td>75821</td></tr><tr><td>9</td><td>35973</td></tr><tr><td>1</td><td>79916</td></tr><tr><td>2</td><td>7845</td></tr></tbody></table></div>"
+      ],
+      "text/plain": [
+       "shape: (4, 2)\n",
+       "┌─────────┬────────────────┐\n",
+       "│ ILOSTAT ┆ Gaussian Noise │\n",
+       "│ ---     ┆ ---            │\n",
+       "│ i64     ┆ u32            │\n",
+       "╞═════════╪════════════════╡\n",
+       "│ 3       ┆ 75821          │\n",
+       "│ 9       ┆ 35973          │\n",
+       "│ 1       ┆ 79916          │\n",
+       "│ 2       ┆ 7845           │\n",
+       "└─────────┴────────────────┘"
+      ]
+     },
+     "execution_count": 9,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "context = dp.Context.compositor(\n",
+    "    data=df,\n",
+    "    privacy_unit=dp.unit_of(contributions=36),\n",
+    "    privacy_loss=dp.loss_of(rho=0.5),\n",
+    "    split_evenly_over=10,\n",
+    "    margins={\n",
+    "        (\"ILOSTAT\", ): dp.Margin(public_info=\"keys\", max_partition_length=60_000_000),\n",
+    "        (\"HWUSUAL\", ): dp.Margin(public_info=\"keys\", max_partition_length=60_000_000),\n",
+    "        (): dp.Margin(max_partition_length=60_000_000),\n",
+    "    },\n",
+    ")\n",
+    "\n",
+    "gaussian_values = (context.query().group_by(\"ILOSTAT\").agg(pl.len().dp.gaussian().alias(\"Gaussian Noise\"))).release().collect()\n",
+    "gaussian_values"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Comparing Results"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/var/folders/dg/nlwnz_jj7rsblr2tzwj962b40000gn/T/ipykernel_68933/709051719.py:11: DeprecationWarning: `apply` is deprecated. It has been renamed to `map_elements`.\n",
```

Comment:
Repeating a comment made elsewhere, we want notebooks to run without warnings... and if that's not possible, we should comment to explain why, and have an issue.